### PR TITLE
Robot ssh key setup alias

### PR DESCRIPTION
**Pull request by erelson**

### CODE REVIEW

File: setup.bash [hunk 1]
```diff
@@ -38,3 +38,19 @@ sfr () {
         ssh freight$@
     fi;
 }
+
+kf () {
+    if [[ $@ && $FETCH_USER ]]; then
```
Comment (xeroxcat) [inline setup.bash:43]: is the idea with $@ that you can ssh-copy-id to many robots at once?

Comment (xeroxcat) [reply setup.bash:43]: if so, worth documenting

Comment (erelson) [reply setup.bash:43]: Oh good point. I just copied it from the other functions, but I had forgotten that you can do `sfr 1234 whoami` type stuff. Looks like that's not applicable for ssh-copy-id.

But I could get fancy and allow support of flags to ssh-copy-id via:

```
ssh-copy-id ${@: 1:$(($# - 1))} $FETCH_USER@fetch${@: -1}
             ^-- args 1 to (n-1) if n > 1         ^-- last arg
```
(I had to google that trickery obviously)

Comment (xeroxcat) [reply setup.bash:43]: oh  yea that would be a good feature i'm sometimes explicitly passing a key with ssh-copy-id

**Pull request merged**